src/core/qgsconditionalstyle.cpp

```diff
@@ -24,11 +24,22 @@ QgsConditionalLayerStyles::QgsConditionalLayerStyles()
   : mRowStyles( QList<QgsConditionalStyle>() )
 {}
 
-QList<QgsConditionalStyle> QgsConditionalLayerStyles::rowStyles()
+QgsConditionalLayerStyles::QgsConditionalLayerStyles( const QgsConditionalLayerStyles &styles )
```

Comment:
Why is this needed? As far as I can see it's doing the same as the default copy constructor would